python/pyspark/sql/functions.py

```diff
@@ -2448,6 +2448,8 @@ def udf(f=None, returnType=StringType()):
 @since(2.3)
 def pandas_udf(f=None, returnType=None, functionType=None):
     """
+    .. note:: Experimental
```

Comment:
not a big deal but can we put this right before doctest just for consistency?

Reply:
yeah, it was hard to find a consistent place to put it.  I changed this one to be after the param definitions, before doctest.  hopefully that's better.